.github/workflows/bundled-size.yaml

```diff
@@ -12,3 +12,7 @@ jobs:
         with:
           build-script: "build-rollup"
           compression: "none"
+          # no need to report on files that aren't the interesting ones
+          exclude: "{./dist/es.js,./dist/module.js,./dist/array.full.js,**/*.map,**/node_modules/**}"
+          # no need to report on very small byte changes which show as 0% change
+          minimum-change-threshold: 300
```

Comment:
a secondary benefit of this reporting is ensuring all changes you're doing are bundled correctly, not just when size bloats up too much, so I'd like to not have this minimum threshold.

"I updated module types, but oh, the module size didn't change - what's going wrong?" oh, gotta export it too 🤷

Reply:
#til 👍 

I've never once looked at them 🙉